source/agora/network/VibeManager.d

```diff
@@ -59,13 +59,23 @@ shared static this ()
 /// And implementation of `agora.network.Manager : NetworkManager` using Vibe.d
 public final class VibeNetworkManager : NetworkManager
 {
+    private RPCClient!(agora.api.Validator.API)[] rpc_clients;
```

Comment:
Cant we just `close()` the socket in `RPCClient` dtor?

Reply:
I think all interface and connections should be closed before invoking node's shutdown procedure. But this might cause holding an unnecessary reference for cases where a connection [is banned ](https://github.com/bosagora/agora/blob/03f36b7f6103396707cde00906226a68ab43d9c5/source/agora/network/Client.d#L657). I will think a better approach.

Reply:
I've updated it with new approach, ready to review